main.go

```diff
@@ -126,7 +130,7 @@ func main() {
 		// Set virtcontainers logger.
 		vc.SetLog(ccLog)
 
-		runtimeConfig, err := loadConfiguration("")
+		runtimeConfig, err := loadConfiguration(context.GlobalString("config"))
```

Comment:
Just a question here. By default, if we called this binary without any `--config` configuration file provided, the call to `context.GlobalString("config")` will return `""`, right ?